File: src/main/java/no/nav/veilarboppfolging/controller/response/OppfolgingPeriodeDTO.java
package no.nav.veilarboppfolging.controller.response;

import java.time.ZonedDateTime;
import java.util.List;
import java.util.UUID;

public record OppfolgingPeriodeDTO(
    UUID uuid,
    String aktorId,
    String veileder,
    ZonedDateTime startDato,
    ZonedDateTime sluttDato,
    String begrunnelse,
    List<KvpPeriodeDTO> kvpPerioder
){}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/response/OppfolgingPeriodeDTO.kt
package no.nav.veilarboppfolging.controller.response

import java.time.ZonedDateTime
import java.util.*

data class OppfolgingPeriodeDTO(
    val uuid: UUID,
    val aktorId: String,
    val veileder: String?,
    val startDato: ZonedDateTime,
    val sluttDato: ZonedDateTime?,
    val begrunnelse: String?,
    val kvpPerioder: List<KvpPeriodeDTO>
)
